extensions/conventional-comments/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "conventional-comments",
+  "title": "Conventional comments",
+  "description": "Paste conventional comment into your foremost application",
+  "icon": "icon.png",
+  "author": "krzysztofzuraw",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Conventional comments",
+      "subtitle": "",
+      "description": "Form to submit conventional comments",
```

Comment:
**suggestion**: Update description
How about something like "Easily insert conventional comments into the frontmost application."